ts/Dashboards/Accessibility/DashboardsAccessibility.ts

```diff
@@ -0,0 +1,88 @@
+/* *
+ *
+ *  (c) 2009 - 2023 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Sebastian Bochan
+ *
+ * */
+
+'use strict';
+
+import Dashboard from '../Dashboard.js';
+
+/* *
+ *
+ *  Imports
+ *
+ * */
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+
+/* *
+ *
+ *  Class
+ *
+ * */
+
+class DashboardsAccessibility {
+    /* *
+    *
+    *  Constructor
+    *
+    * */
+
+    public constructor(
+        dashboard: Dashboard
+    ) {
+        this.dashboard = dashboard;
+
+        this.addTabIndexToCells();
```

Comment:
I presume this module might get bigger and bigger. What do you think of creating a public method `init` where all extra methods like `addTabIndexToCells` will be invoked?

Reply:
Your are right, at this moment its more as a PoC of a class that will be developed and redesign when we will add support for the Edit Mode and other parts.